test/PolynyaIntegration_test.py

```diff
@@ -0,0 +1,185 @@
+import os
+import subprocess
+import sys
+import unittest
+
+import netCDF4
+import numpy as np
+
+
+class Polynya(unittest.TestCase):
```

Comment:
It'd be useful to have a docstring that gives an overview of what the test case is. It doesn't necessarily need to go into details, but could provide a link to where to get more info. e.g., the paper referenced elsewhere.